### PR TITLE
Dx 1734

### PR DESCRIPTION
**Pull request by ajrice6713**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-dx-1734.s3-website-us-east-1.amazonaws.com/index.html

**Review by hamermike — APPROVED** [review]

File: emergency-services/dashboard/about.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% raw %}
```
Comment (jmulford-bw) [inline emergency-services/dashboard/about.md:1]: Empty doc?

**Review by jmulford-bw — APPROVED** [review]
Just had 1 doc that confused me, otherwise 👍

File: emergency-services/dashboard/about.md (hunk 1 above)
Comment (ajrice6713) [reply emergency-services/dashboard/about.md:1]: Jacob and I discussed offline but commenting here for consistency. 

For the sidebar links to work, the gitbook plugin requires them to be linked to a live page. I wanted a Dashboard and Dash distinction in that sidebar to make it easy for customer to navigate the 2 sets of 911 api documentation, and to do that I needed a Dashboard and Dash landing page.

Awaiting a response from the 911 product teams for some guidance on what the content of these pages should be .

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-dx-1734.s3-website-us-east-1.amazonaws.com/index.html

**Pull request merged**